internal/service/provider/file.go

```diff
@@ -18,16 +19,42 @@ type fileParser interface {
 	Do(payload []byte) []byte
 }
 
+// FileHelpersC contain the implemenation of helpers for providers
+type FileHelpersC struct{}
```

Comment:
This struct is not used outside the `provider` package, you could make it private. The test is using it, but the test have access to private structs if needed.

Reply:
Right, and actually I realized I was instantiating the wrong variable. Fixed.